docs/testing_before_pr_checklist.md

```diff
@@ -0,0 +1,22 @@
+For each PR, make sure each of these works
+
+# New items
+* [ ] Make a new project and make sure you can edit in Robot and TeleOp
+* [ ] Add an opmode "Auto" from the + next to Tabs
+* [ ] Add a mechanism "Arm" from the + next to Tabs
+# Using robot
+* [ ] Add a component to the Robot
+* [ ] Add a mechanism to the Robot
+* [ ] Make sure component and Mechanism show up in toolbox for Opmode
+# Events
+* [ ] Add an event to the Mechanism
+* [ ] In Robot, make sure you can see the event
+* [ ] In Opmode, make sure you can see the event
+# Mechanims
+* [ ] Add a public component to the mechanism
+* [ ] Add a private component to the mechanism
+* [ ] Make sure that in the Robot you can see the public component (and not the private one)
+# Code Generation
+* [ ] Generate code (right now "Deploy") and make sure there are no errors
```

Comment:
Add "in the console".
Add another step:
[ ] Check that the deploy zip file contains robot.py, teleop.py, auto.py, and arm.py.